.github/workflows/nlu-hyperopt.yml

```diff
@@ -8,12 +8,19 @@ jobs:
     outputs:
       best_config: ${{ steps.nluhyperopt.outputs.best_config }}
     steps:
-    - uses: actions/checkout@v2
-    - uses: RasaHQ/nlu-hyperopt@v1
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.8.0
+      with:
+        access_token: ${{ github.token }}
+
+    - name: Checkout pull request HEAD commit instead of merge commit
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - uses: ./
```

Comment:
Potentially the `README.md` needs to be updated to align with these workflow changes?

Reply:
You're right, I bumped the version (release will happen on merge of this PR) and updated the default data dir there.